deployment/config.go

```diff
@@ -48,6 +50,14 @@ type Config struct {
 	LogSettings         logger.Settings
 }
 
+type ElasticSearchSettings struct {
+	Enable       bool   // Elasticsearch is going to be created
```

Comment:
This is a very small thing. So feel free to ignore. But all comments here end with a period. So might be good to keep that consistency.

Reply:
Done